src/applications/letters/containers/LetterList.jsx

```diff
@@ -128,50 +128,54 @@ export class LetterList extends React.Component {
         {eligibilityMessage}
 
         <br />
+        <h3 slot="headline">Other sources of VA benefit documentation</h3>
         <p>
           A lot of people come to this page looking for their Post-9/11 GI Bill
           statement of benefits, their Certificate of Eligibility (COE) for home
           loan benefits, and their DD214. We don’t have these documents
           available here yet, but if you’re eligible for them, you can get them
           through these links:
         </p>
-        <ul className="vads-u-margin-bottom--9">
+        <ul
+          className="vads-u-margin-bottom--9"
+          style={{ listStyleType: 'disc' }}
+        >
           <li>
-            <a href="/education/download-letters/" target="_blank">
-              <strong>Download your VA education letters.</strong>
+            <a
+              href="/education/download-letters/"
+              target="_blank"
+              style={{ textDecoration: 'none' }}
+            >
+              <p>VA education letters</p>
             </a>
           </li>
           <li>
             <a
               href="/education/gi-bill/post-9-11/ch-33-benefit"
               target="_blank"
+              style={{ textDecoration: 'none' }}
```

Comment:
Could you use the VA design system util class `.vads-u-text-decoration--none` for these links instead of an inline style? https://design.va.gov/foundation/utilities/text-decoration